src/stats.rs

```diff
@@ -4,3 +4,15 @@ pub struct Stats {
     pub dropped_by_kernel: u32,
     pub dropped_by_interface: u32,
 }
+
+pub const EMPTY_STATS: Stats = Stats{ received: 0, dropped_by_kernel: 0, dropped_by_interface: 0 };
```

Comment:
The Idiomatic rust way to do this is to implement `Default` for Stats.

Reply:
👍